src/ansys/templates/python/pyansys/README.rst

```diff
@@ -0,0 +1,130 @@
+Py{{ cookiecutter.product_name }} {{ cookiecutter.library_name }}
+{{ '=' * (cookiecutter.__project_name_slug | length) }}
+
+{{ cookiecutter.short_description }}
+
+
+How to install
+--------------
+
+At least two installation modes are provided: user and developer.
+
+For users
+^^^^^^^^^
+
+User installation can be performed by running:
+
+.. code:: bash
+
+    python -m pip install {{ cookiecutter.__pkg_name }}
+
+For developers
+^^^^^^^^^^^^^^
+
+Installing Py{{ cookiecutter.product_name }} {{ cookiecutter.library_name }} in developer mode allows
+you to modify the source and enhance it.
+
+Before contributing to the project, please refer to the `PyAnsys Developer's guide`_. You will 
+need to follow these steps:
+
+1. Start by cloning this repository:
+
+    .. code:: bash
+
+        git clone {{ cookiecutter.repository_url }}
+
+2. Create a fresh-clean Python environment and activate it. Refer to the
+   official `venv` documentation if you require further information:
+
+    .. code:: bash
+
+        # Create a virtual environment
+        python -m venv .venv
+
+        # Activate it in a Linux environment
+        python -m venv .venv && source .venv/bin/activate
+
+        # Activate it in a Windows CMD environment
+        source .venv\Scripts\activate.bat
+
+        # Activate it in a Windows Powershell environment
+        source .venv\Scripts\Activate.ps1
+
+
+3. Make sure you have the latest version of `pip`_:
+
+    .. code:: bash
+
+        python -m pip install -U pip
+
+4. Install the project in editable mode:
+
+    .. code:: bash
+    
+        python -m pip install --editable {{ cookiecutter.__pkg_name }}
+
+5. Install additional requirements (if needed):
+
+     .. code:: bash
+
+        python -m pip install -r requirements/requirements_build.txt
+        python -m pip install -r requirements/requirements_doc.txt
+        python -m pip install -r requirements/requirements_tests.txt
+
+
+6. Finally, verify your development installation by running:
+
+    .. code:: bash
+        
+        python -m pip install -r requirements/requirements_tests.txt
+        pytest tests -vvv
```

Comment:
Using higher verbosity levels (-vvv, -vvvv, …) is supported, but has no effect in pytest itself at the moment, however some plugins might make use of higher verbosity.

from:
https://docs.pytest.org/en/latest/how-to/output.html

```suggestion
        pytest tests -v
```